source/tests/base_user_types.py

```diff
@@ -60,13 +65,12 @@ def browse_user_guide(self):
 
     @task
     def register_user(self):
-        """ Register this user as a new user account. """
+        """Register this user as a new user account."""
         if SERVE_LOCUST_DO_CREATE_OBJECTS is False or SERVE_LOCUST_DO_CREATE_OBJECTS == "False":
```

Comment:
Why not like this?

Consider comment above

```suggestion
        if not SERVE_LOCUST_DO_CREATE_OBJECTS:
```